### ISSUE TITLE
Discussion autour du terme "Neutre"

### ISSUE THREAD

**Issue by rcaze** [rcaze/Charte#1]
Et pourquoi pas "Libre de tous parti politique". Je trouve que ça sonne bien :)

**Comment by AmbreM** [comment]
Je suis d'accord, il est beaucoup mieux d'être "pour" quelque-chose (e.g.: libre de...) que "contre". Le vocabulaire positif est important et ça change la perspective qu'on donne au mouvement.
